spec/ddtrace/contrib/presto/client_spec.rb

```diff
@@ -24,7 +24,7 @@
   end
   let(:service) { 'presto' }
   let(:host) { ENV.fetch('TEST_PRESTO_HOST', 'localhost') }
-  let(:port) { ENV.fetch('TEST_PRESTO_PORT', 8080) }
+  let(:port) { ENV.fetch('TEST_PRESTO_PORT', 8080).to_i }
```

Comment:
Just to doublecheck: Why is this needed now but was not needed before?

Reply:
I don't know.

Reply:
I guess the library works with both an Integer as well as a String, so this seems harmless, I just got curious about this change because otherwise it didn't look like we had changed anything that would require this.